src/Governance.sol

```diff
@@ -467,7 +467,10 @@ contract Governance is MultiDelegateCall, UserProxyFactory, ReentrancyGuard, Own
         }
 
         // == Unregister Condition == //
-        // e.g. if `UNREGISTRATION_AFTER_EPOCHS` is 4, the 4th epoch flip that would result in SKIP, will result in the initiative being `UNREGISTERABLE`
+        // e.g. if `UNREGISTRATION_AFTER_EPOCHS` is 4, the initiative will become unregisterable after spending 4 epochs
+        // while being in one of the following conditions:
+        //  - in `SKIP` state (not having received enough votes to cross the voting threshold)
+        //  - in `CLAIMABLE` state (having received enough votes to cross the voting threshold) but never being claimed
```

Comment:
This is actually fixing Dedaub H3, right?

Reply:
If we consider this comment the new specification, then yes 😄